sources/tests/docker-compose.yml

```diff
@@ -0,0 +1,207 @@
+version: "3"
+
+# Services
+services:
+  # API
+  api:
+    image: api-${PROJECT_NAME}:${BUILD_REF}-${BUILD_ID}
+    container_name: api-${PROJECT_NAME}-${BUILD_REF}-${BUILD_ID}
+    build:
+      context: ../packages/backend
+      dockerfile: ./apps/api/Dockerfile.dev
+    environment:
+      - NODE_ENV=docker
+      - ENVIRONMENT=local
+      - PORT=${API_PORT}
+      - POSTGRES_HOST=postgres
+      - POSTGRES_PORT=${POSTGRES_PORT}
+      - POSTGRES_DB=${POSTGRES_DB}
+      - POSTGRES_USER=${POSTGRES_USER}
+      - POSTGRES_PASSWORD=${POSTGRES_PASSWORD}
+      - REDIS_HOST=redis
+      - REDIS_PORT=${REDIS_PORT}
+      - REDIS_PASSWORD=${REDIS_PASSWORD}
+      - REDIS_STANDALONE_MODE=${REDIS_STANDALONE_MODE}
+      - QUEUE_PREFIX=${QUEUE_PREFIX}
+      - KEYCLOAK_AUTH_URL=${KEYCLOAK_AUTH_URL}
+      - KEYCLOAK_REALM=${KEYCLOAK_REALM}
+      - KEYCLOAK_CLIENT_STUDENT=${KEYCLOAK_CLIENT_STUDENT}
+      - KEYCLOAK_CLIENT_INSTITUTION=${KEYCLOAK_CLIENT_INSTITUTION}
+      - KEYCLOAK_CLIENT_AEST=${KEYCLOAK_CLIENT_AEST}
+      - KEYCLOAK_CLIENT_SUPPORTING_USERS=${KEYCLOAK_CLIENT_SUPPORTING_USERS}
+      - DB_SCHEMA=${DB_SCHEMA}
+      - E2E_TEST_STUDENT_USERNAME=${E2E_TEST_STUDENT_USERNAME}
+      - E2E_TEST_STUDENT_PASSWORD=${E2E_TEST_STUDENT_PASSWORD}
+      - E2E_TEST_BCeID_USERNAME=${E2E_TEST_BCeID_USERNAME}
+      - E2E_TEST_BCeID_PASSWORD=${E2E_TEST_BCeID_PASSWORD}
+      - BCeID_WEB_SERVICE_WSDL=${BCeID_WEB_SERVICE_WSDL}
+      - BCeID_WEB_SERVICE_ONLINE_SERVICE_ID=${BCeID_WEB_SERVICE_ONLINE_SERVICE_ID}
+      - BCeID_WEB_SERVICE_REQUESTER_USER_GUID=${BCeID_WEB_SERVICE_REQUESTER_USER_GUID}
+      - BCeID_WEB_SERVICE_AUTH_USER_NAME=${BCeID_WEB_SERVICE_AUTH_USER_NAME}
+      - BCeID_WEB_SERVICE_AUTH_USER_PASSWORD=${BCeID_WEB_SERVICE_AUTH_USER_PASSWORD}
+      - WORKFLOW_SA_USER_NAME=${WORKFLOW_SA_USER_NAME}
+      - WORKFLOW_SA_PASSWORD=${WORKFLOW_SA_PASSWORD}
+      - FORMS_URL=http://formio:3001
+      - FORMS_SA_USER_NAME=${FORMS_SA_USER_NAME}
+      - FORMS_SA_PASSWORD=${FORMS_SA_PASSWORD}
+      - SIMS_API_CLIENT_ID=${SIMS_API_CLIENT_ID}
+      - SIMS_API_CLIENT_SECRET=${SIMS_API_CLIENT_SECRET}
+      - DUMMY_BCeID_ACCOUNT_RESPONSE=${DUMMY_BCeID_ACCOUNT_RESPONSE}
+      - SITE_MINDER_LOGOUT_URL=${SITE_MINDER_LOGOUT_URL}
+      - ATBC_LOGIN_ENDPOINT=${ATBC_LOGIN_ENDPOINT}
+      - ATBC_USERNAME=${ATBC_USERNAME}
+      - ATBC_PASSWORD=${ATBC_PASSWORD}
+      - ATBC_APP=${ATBC_APP}
+      - ATBC_ENDPOINT=${ATBC_ENDPOINT}
+      - FILE_UPLOAD_MAX_FILE_SIZE=${FILE_UPLOAD_MAX_FILE_SIZE}
+      - FILE_UPLOAD_ALLOWED_EXTENSIONS=${FILE_UPLOAD_ALLOWED_EXTENSIONS}
+      - BYPASS_APPLICATION_SUBMIT_VALIDATIONS=${BYPASS_APPLICATION_SUBMIT_VALIDATIONS}
+      - SWAGGER_ENABLED=${SWAGGER_ENABLED}
+      - E2E_TEST_AEST_BUSINESS_ADMINISTRATORS_USER=${E2E_TEST_AEST_BUSINESS_ADMINISTRATORS_USER}
```

Comment:
The E2E_TEST_AEST_*  can be added to env-example as a suggestion.

Reply:
Or should it be added to .env-e2e-tests?

Reply:
We need those also to run the tests locally (e.g. `npm run test:e2e:api`) and debug them also. During development, it is pretty useful.
Besides that, those credentials should be considered sensitive info because they are real Keycloak users, even being used only for the E2E tests they would be able to access our DEV env.